circus-train-avro/src/main/java/com/hotels/bdp/circustrain/avro/util/SchemaCopier.java

```diff
@@ -13,35 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.hotels.bdp.circustrain.avro.util;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import static com.hotels.bdp.circustrain.avro.util.AvroStringUtils.fileName;
 
-import java.io.IOException;
-import java.nio.file.Files;
+package com.hotels.bdp.circustrain.avro.util;
 
+import com.hotels.bdp.circustrain.api.CircusTrainException;
+import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
-import org.apache.hadoop.hive.conf.HiveConf;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import com.hotels.bdp.circustrain.api.CircusTrainException;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
Your imports don't match our standard import ordering, statics need to come first etc. Please review and make sure they line up with what's documented on our "London standards" page. Nothing should change here other than what you've added.